src/group_theory/subgroup/basic.lean

```diff
@@ -1476,6 +1476,10 @@ iff.rfl
 @[to_additive] lemma range_eq_map (f : G →* N) : f.range = (⊤ : subgroup G).map f :=
 by ext; simp
 
+@[to_additive]
+lemma range_eq_top_of_surjective {f : G →* N} (hf : function.surjective f) :
```

Comment:
This seems to be a duplicate of `range_top_iff_surjective`, just 22 lines lower down!

Reply:
🤦🏻 . I wonder why I didn’t find it. I really miss `find_theorems`